colanderalchemy/schema.py

```diff
@@ -113,6 +114,11 @@ class MyModel(Base):
            method of this instance.
 
            Default: 'ignore'
+        parents\_
+           An ``SQLAlchemySchemaNode`` parent list to avoid relationship circular
+           dependencies and thus prevent infinite recursion. Used internally.
+
+           Default: []
```

Comment:
Should we be documenting an argument that shouldn't really be used externally?  Is there any use for this externally?